src/SFA.DAS.Commitments.Api.UnitTests/Orchestrators/Mappers/WhenMappingACommitmentView.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Commitments.Api.Orchestrators.Mappers;
+using SFA.DAS.Commitments.Application.Rules;
+using SFA.DAS.Commitments.Domain;
+using SFA.DAS.Commitments.Domain.Entities;
+
+namespace SFA.DAS.Commitments.Api.UnitTests.Orchestrators.Mappers
+{
+    [TestFixture]
+    public class WhenMappingACommitmentView
```

Comment:
Probably outside of the scope of this story to change it, but CommitmentView is not a very descriptive name.